ui/orbital.c

```diff
@@ -535,24 +539,17 @@ static void* orbital_display_main(void* arg)
         ORBITAL_WIDTH, ORBITAL_HEIGHT, flags);
 
     count = 0;
-    if (!SDL_Vulkan_GetInstanceExtensions(ui.sdl_window, &count, NULL)) {
-        error_report("SDL_Vulkan_GetInstanceExtensions returned false");
-        return NULL;
-    }
+    SDL_Vulkan_GetInstanceExtensions(ui.sdl_window, &count, NULL);
     if (count == 0) {
         error_report("SDL_Vulkan_GetInstanceExtensions failed");
     }
 
-    const char **extensionNames = (const char **)malloc((count + 3) * sizeof(char*));
+    const char **extensionNames = (const char **)malloc((count + 2) * sizeof(char*));
     extensionNames[count + 0] = VK_KHR_SURFACE_EXTENSION_NAME;
     extensionNames[count + 1] = VK_EXT_DEBUG_UTILS_EXTENSION_NAME;
-    extensionNames[count + 2] = VK_EXT_DEBUG_REPORT_EXTENSION_NAME;
```

Comment:
Why is this extension removed? It was added in:
https://github.com/AlexAltea/orbital-qemu/commit/fca99f2c5320333862639e2e8425cfca4483c19a

If it causes issues in some systems it should be conditionally disabled.